addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -2297,6 +2353,101 @@ X[]
                     });
                 });
             });
+            describe('Nested editable zone (inside contenteditable=false element)', () => {
+                it('should remove the uneditable nesting zone if the last element of the nested zone is empty', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: unformat(`
+                            <div contenteditable="false">
+                                <div contenteditable="true">
+                                    <p>[]<br></p>
+                                </div>
+                            </div>
+                        `),
+                        stepFunction: deleteBackward,
+                        contentAfter: unformat(`
+                            <p>[]<br></p>
+                        `),
+                    });
+                });
+                it('should not remove the last element of a nested zone if not empty', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: unformat(`
+                            <div contenteditable="false">
+                                <div contenteditable="true">
+                                    <p>[]content</p>
+                                </div>
+                            </div>
+                        `),
+                        stepFunction: deleteBackward,
+                        contentAfter: unformat(`
+                            <div contenteditable="false">
+                                <div contenteditable="true">
+                                    <p>[]content</p>
+                                </div>
+                            </div>
+                        `),
+                    });
+                });
```

Comment:
It's even weirder that backspace doesn't have the same effect just because the node in which you do backspace isn't empty.

EDIT: I had a second look at the spec and I feel like the result of this test should be `<p>content</p>` instead. What do you think ?